<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>581</width>
      <height>491</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>579</width>
       <height>489</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>591</width>
        <height>491</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeWidget {
	background-color: #000000;

}</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Folder</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Usage (%)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size (MB)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contents</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>29</y>
      <width>601</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="diskDisplay">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionScan"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScan"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/pngfuel.com.png</normaloff>:/icons/pngfuel.com.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/pngfuel.com (1).png</normaloff>:/icons/pngfuel.com (1).png</iconset>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/pngfuel.com (2).png</normaloff>:/icons/pngfuel.com (2).png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionEnter_Full_Screen">
   <property name="text">
    <string>Enter Full Screen</string>
   </property>
  </action>
  <action name="actionExit_Full_Screen">
   <property name="text">
    <string>Exit Full Screen</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
